Makefile

```diff
@@ -8,8 +8,18 @@ test: bats
 	PATH="./bats/bin:$$PATH" test/run
 	cd plugins/python-build && $(PWD)/bats/bin/bats $${CI:+--tap} test
 
-test-build: bats
-	cd plugins/python-build && $(PWD)/bats/bin/bats $${CI:+--tap} test/build
+PYTHON_BUILD_ROOT := $(CURDIR)/plugins/python-build
```

Comment:
Doing this in make instead of bats seems a bit harder to extend in the future. Imagine all the open issues we have where the failed build is only evident after testing the compiled result.